File: ReadLine-tst/.cpp
//1
#include <iostream>
using namespace std;

int main()
{
    int a, b;
    cin >> a >> b;
    cout << a + b << endl;
    return 0;
}





// 2
#include <iostream>
using namespace std;

int main()
{
    int ans = 0, n;
    cin >> n;
    for(int i = 0; i < n; i++)
    {
        int a;
        cin >> a;
        ans += a;
    }
    cout << ans << endl;
    return 0;
}

File: ReadLine-tst/.py
# Python2
# 1
s = raw_input()
a, b = s.split(" ")
print int(a) + int(b)


# 2
ans = 0
n = int(raw_input())
str = raw_input().split(" ")
for i in range(n) :
    ans = ans + int(str[i])
print ans




# Python3
# 1
s = input()
a, b = s.split(" ")
print(int(a) + int(b))



# 2
ans = 0
n = int(input())
str = input().split(" ")
for i in range(n) :
    ans = ans + int(str[i])
print(ans)